lib/utils/validation_reporters.ts

```diff
@@ -14,6 +14,10 @@ function getContextPath(context: Context): string {
   const keysPath = context.map(({ key }) => key).join(".");
   const lastType = context[context.length - 1].type;
 
+  if ("never" === lastType.name) {
```

Comment:
perhaps add a [test case](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/utils/__tests__/validation_reporters.test.ts) for this

Reply:
done